src/hooks/item.ts

```diff
@@ -183,18 +186,18 @@ export default (
 
       const { data: currentMember } = useCurrentMember();
       itemWsHooks?.useSharedItemsUpdates(
-        getUpdates ? (currentMember as Record<Member>)?.get('id') : null,
+        getUpdates ? (currentMember as RecordOf<Member>)?.id : null,
```

Comment:
I think you can define useCurrentMember return type to void casting the type. I think the secret is to define the type as follow:
```
return useQuery({
        queryKey: buildAppContextKey(itemId),
        queryFn: (): Promise<RecordOf<AppContext>> => {
          return Api.getContext({
            itemId,
            token,
            apiHost,
          }).then((data) => AppContextRecord(data));
        },
```
Basile did it a lot for apps-query-client, you might ask him.